{
    NDT_sig_X: 0.001, NDT_sig_Y: 0.001, NDT_sig_Yaw: 0.001, 
    Pred_a1: 0.001, Pred_a2: 0.005, Pred_a3: 1.0e-05, Pred_a4: 0.0005, 
    init_sig_x: 0.01, init_sig_y: 0.001, init_sig_yaw: 0.001,
    init_x : 0.0, init_y : 0.0, init_yaw : 0.0
}
